.travis/stages/verify/check-links-and-yaml-syntax/check-links

```diff
@@ -15,28 +17,31 @@ fi
 
 function checkUri() {
   local -r uri="$1"
-  
-  local -r curlResult=$(curl -s --head "$uri" | head -n 1)
-  echo "$uri -- $curlResult"  
-  # Temporarily disabled. We're unable to connect to A&A.org from Travis.
-
-  # echo $curlResult |  grep -qE  "HTTP/2 200|(200|302) (OK|Found)" \
-  #  || reportBrokenUri "$uri"
+  local output
+  if ! output="$FAILURE_PREFIX$(curl -Lsf -o /dev/null -w "%{http_code}" "$uri")"; then
+    reportBrokenUri "$uri" "$output"
+    FAILURE=1
+  fi
 }
 
 function reportBrokenUri() {
-  FAILURE=1 
   local -r red="\e[38;5;196m"
   local -r blue="\e[38;5;75m"
-  echo -en "Error: ${red}$uri${end} ${blue}"
-  curl -s --head "$uri" | head -n 1 | sed 's|HTTP/[12][.01]?||'
```

Comment:
What surprised me here is that `$uri` is not actually the parameter passed to this function, (which was unused before my change), but instead the global variable that is defined below at `while read -r uri; do`.

I tried limiting the scope of this loop, but without success, so `$uri` could still  be used here.
I wonder if it's better to use the global uri everywhere, or just ignore it an pass it everywhere explicitly 🤔

Reply:
Fortunate bug, I think it was probably intended to assign it to a local function variable and then use that variable. The variables name happened to match so it worked :disappointed: